### DISCUSSION TITLE
How to use math in raw html block

### DISCUSSION THREAD

**Discussion by stuvjordan-uroc** [quarto-dev/quarto-cli#10762] [Q&A] [answered]
### Reproducible example in git repo here:

[https://github.com/stuvjordan-uroc/rawhtmlwithmath](https://github.com/stuvjordan-uroc/rawhtmlwithmath)

### Description

Note: this question might seem to be the same as [this one](https://github.com/quarto-dev/quarto-cli/discussions/10041), but it's not.  The problem that generated the question in that post was solved by not using a raw html block at all.  In contrast, I have to use a raw html block for what I'm doing, so the answer offered in that discussion does not address my question.

In a website that I'm writing with quarto, I need to include some tables with some custom styling.  My approach so far is write the tables in raw html blocks, adding my own css classes, and styling those classes in a `styles.css` at the root of my project.  

I want to add some nicely-typesetted math to my custom tables.

How do I do that?

Part of the accepted answer to [this discussion question](https://github.com/quarto-dev/quarto-cli/discussions/10041) made me think that I might get latex markup in my raw html tables rendered into pretty math if I put the markup in delimiters like this `\( \)`.  But that's not working in the code in the github repo I've linked above.

So...how do I make latex markup in a raw html block rendered to pretty math?

**Comment by mcanouil** [comment]
Could you make your example reproducible? Right now it's only an HTML table.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by stuvjordan-uroc** [reply]
Thanks, @mcanouil !!!

I've created a reproducible git repo and edited my post accordingly.

**Reply by mcanouil** [reply, marked as answer]
Thanks, I was able to play around with it.
What you need is to use the following that will tell Quarto that some content is actually markdown and needs parsing.

```
<span data-qmd="$$\sum_{n=0}^\infty \frac{1}{n^2}$$"></span>
```

This is documented in https://quarto.org/docs/authoring/tables.html#html-tables

**Reply by stuvjordan-uroc** [reply]
Ah!!! Totally missed that part of the table docs.  Thanks!!!
